src/platform/site-wide/user-nav/components/PageNotFound.jsx

```diff
@@ -15,6 +15,12 @@ const PageNotFound = ({ recordEvent = recordEventFn } = {}) => {
     [recordEvent],
   );
 
+  useEffect(() => {
+    // Hide the breadcrumbs.
+    const breadcrumbs = document.getElementsByClassName('va-nav-breadcrumbs');
```

Comment:
Note that this hides the Drupal based breadcrumbs as well as React code based ones.

Reply:
This makes sense for hiding Drupal controlled breadcrumbs but I'm curious if its possible to rely on state & conditional rendering for hiding the React based breadcrumbs?

Reply:
We can do it that way for some apps, but not others. We would have to use that `hasPageNotFound()` check on the apps that create their own breadcrumbs, but we will have to change the code on each of those apps. Different apps render the breadcrumbs in different ways. Some use the breadcrumbs component, others use the HTML with classes to create them, so there is no one way they are done. Definitely doable, but lots of changes.